recipe/meta.yaml

```diff
@@ -32,8 +32,7 @@ requirements:
 test:
   requires:
     - python <3.10
```

Comment:
It would be nice to update test/requires and add test/commands:
```
test:
  requires:
    - pip
    - python
  commands:
    - pip check
  imports:
    - certifi
```

Reply:
We do not need use `python <3.10` if we build for all platforms. It's only needed if we build `noarch `python package